### PR TITLE
Ruby 💎 capstone group project submission

### PR DESCRIPTION
**Pull request by mahabubx7**
### Ruby 💎 capstone group project submission

What does this PR do?
This PR is raised to implement the following features: 👍 :

 - [x] Add Game, Books, and Music management options
  - [x] Implement catalogs management methods
  - [x] Preserve data in JSON files
  - [x] Add unit tests for all methods
  - [x] Add schema based on the UML
  - [x] Add documentation
  - [x] Use Git-Flow to introduce new features
 - [x] Description of the tasks completed 👇
 - [x] Create catalogs of my things console app
 - [x] Install Ruby gems
 - [x] Added presentation video link in `README.md` 
 
> 💯 Use the linter tool that analyses my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 
How should it be tested
Clone the repository

```sh
git clone https://github.com/ShiroYaksha90/catalog-of-my-things.git
```
Navigate to the folder
```sh
cd catalog-of-my-things
```
Checkout the branch

```sh
git checkout dev
```
Run app
```sh
 ruby main.rb
```

Run tests
```sh
rspec
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,236 +1,113 @@
...
---->
 
-### Deployment
-
-You can deploy this project using:
+### Install
 
-<!--
-Example:
+To run the project, execute the folllowing in a terminal.
```
Comment (tugsem) [inline README.md:70]: - [ ] Kindly add the command to run the project in the console as per the requirement.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @mahabubx7 , @Ouail-01  and @ShiroYaksha90 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️app works as expected
✔️correct use of gitflow
✔️professional documentation & readme
❌ No video presentation link in the readme
✔️ Classes created according to the UML diagram.
✔️ App tested using rspec thoroughly

## Required Changes ♻️
- [x] Please add a zoom video presentation link and describe each of you have done showing the code and highlight a piece of code that you find interesting or proud of making. It should not pass 5 minutes.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mahabubx7) [reply README.md:74]: added changes

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ShiroYaksha90  and @mahabubx7  and @Ouail-01 ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**